File: taskapp-api/src/task/app/TaskNotify.java
package task.app;

import byps.BRemote;
import byps.RemoteException;

/**
 * @BClientRemote
 * @since 1.0.0.1
 */
public interface TaskNotify extends BRemote {

	public int receiveTask(TaskInfo task) throws RemoteException;
	
}

File: taskapp-ser/src-cs/task/app/TaskNotify.cs
using System;
using System.Collections.Generic;
using byps;

namespace task.app
{
	
	// checkpoint byps.gen.cs.GenRemoteClass:93
	public interface TaskNotify : BRemote {
	
		int ReceiveTask(TaskInfo task);
		void ReceiveTask(TaskInfo task, BAsyncResult<int> asyncResult) ;
		
		
	}
}  // end namespace
